packages/workspace/src/browser/workspace-input-dialog.ts

```diff
@@ -54,4 +57,36 @@ export class WorkspaceInputDialog extends SingleTextInputDialog {
         // Add the path and icon div before the `inputField`.
         this.contentNode.insertBefore(element, this.inputField);
     }
+
+    /**
+     * Append the human-readable absolute root `path` to the dialog.
+     */
+    protected appendRootName(): void {
+        if (this.workspaceService && this.workspaceService.isMultiRootWorkspaceOpened) {
+            const roots = this.workspaceService.tryGetRoots();
+            const RootsWithSameName = roots.filter(singleRoot => {
```

Comment:
You can use [some()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) to determine if a root has the same name.